### PR TITLE
fix(coibnasepro) - watch tickers filter

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei

**Comment by ttodua** [comment]
watchTickers was totally broken, didnt even work/return anything

File: ts/src/pro/coinbasepro.ts [hunk 1]
```diff
@@ -775,9 +779,11 @@ export default class coinbasepro extends coinbaseproRest {
             for (let i = 0; i < messageHashes.length; i++) {
                 const currentMessageHash = messageHashes[i];
                 const parts = currentMessageHash.split ('::');
-                const symbolsString = parts[1];
-                const symbols = symbolsString.split (',');
-                if (this.inArray (symbol, symbols)) {
+                const marketIdsString = parts[1];
+                // @ts-ignore
+                const marketIdsStringClean = marketIdsString.replaceAll (':', '');
```
[the remark this answers is not in the document]
Comment (carlosmiei) [reply ts/src/pro/coinbasepro.ts:784]: ok we're already using watchMultiple so this logic is not needed

File: ts/src/pro/coinbasepro.ts [hunk 2]
```diff
@@ -768,19 +770,8 @@ export default class coinbasepro extends coinbaseproRest {
             const ticker = this.parseTicker (message);
             const symbol = ticker['symbol'];
             this.tickers[symbol] = ticker;
-            const type = this.safeString (message, 'type');
-            const messageHash = type + ':' + marketId;
+            const messageHash = 'ticker:' + symbol;
```
Comment (carlosmiei) [inline ts/src/pro/coinbasepro.ts:773]: @ttodua as you can see using the watchMultiple approach, the code inside handleTicker is vastly simplified

**Comment by carlosmiei** [comment]
DEMO

```
p coinbasepro watchTickers '["BTC/USDT", "ETH/USDT", "DOGE/USDT", "ADA/USDT"]'          
Python v3.11.5
CCXT v4.2.7
coinbasepro.watchTickers(['BTC/USDT', 'ETH/USDT', 'DOGE/USDT', 'ADA/USDT'])
{'ETH/USDT': {'ask': 2259.67,
              'askVolume': 7.3e-06,
              'average': 2250.0,
              'baseVolume': 4264.84671294,
              'bid': 2258.75,
              'bidVolume': 0.21336,
              'change': 19.34,
              'close': 2259.67,
              'datetime': '2024-01-08T12:46:35.398Z',
              'high': 2269.35,
              'info': {'best_ask': '2259.67',
                       'best_ask_size': '0.00000730',
                       'best_bid': '2258.75',
                       'best_bid_size': '0.21336000',
                       'high_24h': '2269.35',
                       'last_size': '0.32438752',
                       'low_24h': '2166.75',
                       'open_24h': '2240.33',
                       'price': '2259.67',
                       'product_id': 'ETH-USDT',
                       'sequence': 9844103382,
                       'side': 'buy',
                       'time': '2024-01-08T12:46:35.398494Z',
                       'trade_id': 17440260,
                       'type': 'ticker',
                       'volume_24h': '4264.84671294',
                       'volume_30d': '175985.98940401'},
              'last': 2259.67,
              'low': 2166.75,
              'open': 2240.33,
              'percentage': 0.8632656796097004,
              'previousClose': None,
              'quoteVolume': None,
              'symbol': 'ETH/USDT',
              'timestamp': 1704717995398,
              'vwap': None}}
{'ADA/USDT': {'ask': 0.511,
              'askVolume': 12597.77,
              'average': 0.516,
              'baseVolume': 1403805.63,
              'bid': 0.51,
              'bidVolume': 10633.3,
              'change': -0.012,
              'close': 0.51,
              'datetime': '2024-01-08T12:46:44.687Z',
              'high': 0.526,
              'info': {'best_ask': '0.511',
                       'best_ask_size': '12597.77',
                       'best_bid': '0.510',
                       'best_bid_size': '10633.30',
                       'high_24h': '0.526',
                       'last_size': '3.97',
                       'low_24h': '0.464',
                       'open_24h': '0.522',
                       'price': '0.51',
                       'product_id': 'ADA-USDT',
                       'sequence': 317682102,
                       'side': 'sell',
                       'time': '2024-01-08T12:46:44.687168Z',
                       'trade_id': 589989,
                       'type': 'ticker',
                       'volume_24h': '1403805.63000000',
                       'volume_30d': '30809838.26000000'},
              'last': 0.51,
              'low': 0.464,
              'open': 0.522,
              'percentage': -2.2988505747126435,
              'previousClose': None,
              'quoteVolume': None,
              'symbol': 'ADA/USDT',
              'timestamp': 1704718004687,
              'vwap': None}}
```

**Pull request merged**